ext/storage_sources/gcp_store/README.md

```diff
@@ -0,0 +1,88 @@
+# WiredTiger's GCP Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored
+in Google Cloud Storage using WiredTiger’s provided internal abstraction for storing data in an
+object storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the GCP extension enabled.
+
+### Requirements
+* CMake 3.11 or higher
+* G++ 8.4 or higher
+* Abseil 20200923 Patch 3
```

Comment:
`or higher` here too

Reply:
amended in https://github.com/wiredtiger/wiredtiger/pull/8803/commits/8df3c78d769c02945414dbffcfbedceefc0c8b60